tests/helpers.js

```diff
@@ -3,8 +3,8 @@ const EC = require('elliptic').ec;
 const ec = new EC('secp256k1');
 const signingKey = ec.keyFromPrivate('3d6f54430830d388052865b95c10b4aeb1bbe33c01334cf2cfa8b520062a0ce3');
 
-function createSignedTx(){
-	const txObject = new Transaction(signingKey.getPublic('hex'), 'wallet2', 10);
+function createSignedTx(amount = 10){
```

Comment:
should this really have a default value instead of throwing an Error?

Reply:
Hardcoding doesn't seem like a viable option...

Reply:
@TheDevMinerTV This is a helper function for the unit tests. That's why we hard code (to verify other functionalities). It's not the case in the "production" version.

Reply:
Okay, haven't looked at the file's name...